static/js/settings_users.js

```diff
@@ -317,13 +319,8 @@ function open_user_info_form_modal(person) {
     return user_info_form_modal;
 }
 
-exports.on_load_success = function (realm_people_data) {
-    meta.loaded = true;
-
-    populate_users(realm_people_data);
-
+exports.register_click_handlers = function () {
```

Comment:
Can you clean up this commit series to have steps like extracting `register_click_handlers` as a new function as independent commits?  It'd be more readable that way.